manual/manual/cmds/lexyacc.etex

```diff
@@ -52,8 +52,9 @@ The following command-line options are recognized by "ocamllex".
 \item["-ml"]
 Output code that does not use OCaml's built-in automata
 interpreter. Instead, the automaton is encoded by OCaml functions.
-This option mainly is useful for debugging "ocamllex", using it for
-production lexers is not recommended.
+This option tends to produce faster code (compared to not using "-ml")
```

Comment:
I would remove `(compared to not using "-ml")`.

Reply:
I was afraid that without this, the text could be read as "ocamllex produces faster code with the native compiler than with bytecode".

Reply:
It does not sound confusing to me. Maybe someone else can chime in.

Reply:
Why not say "this option improves performances when using the native compiler, but decreases them when using the bytecode compiler"?

Reply:
P.S.: I think that giving any kind of "typical" number on the performance difference would be useful. Take a reasonable lexer that someone wrote (without the `Location.dummy` trick), measure the speedup and slowdown, and give the numbers: "improves performances (we observed 2x) ... decreases them (we observed 3x) ...". Otherwise there is not enough information for the user to tell whether they should look at it or it's going to be neglectible for their usecase.

Reply:
Text adapted as suggested.

I don't think that many people are sensitive to the performances of both native and bytecode, so saying that -ml is faster in native and slower in bytecode should be enough for most users.

"Typical" numbers would need to be somehow explained (what do we measure exactly, only the time spent in the ocamllex part itself, or also the action part); and performance can greatly depend on the style of the lexer (doing more work in the regexps, vs actions calling themselves recursively).

Anyway, I don't think this is worth spending too much time on that; it's not like ocamllex was a typical bottleneck, nor that `-ml` brought incredible speedups.